pkg/proc/interface.go

```diff
@@ -105,13 +108,17 @@ type ProcessManipulation interface {
 type BreakpointManipulation interface {
 	Breakpoints() *BreakpointMap
 	SetBreakpoint(addr uint64, kind BreakpointKind, cond ast.Expr) (*Breakpoint, error)
+	WriteBreakpointFn() WriteBreakpointFn
 	ClearBreakpoint(addr uint64) (*Breakpoint, error)
 	ClearInternalBreakpoints() error
 }
 
 // CommonProcess contains fields used by this package, common to all
 // implementations of the Process interface.
 type CommonProcess struct {
+	// Path is the path on disk to the executable being debugged.
+	Path string
```

Comment:
This seems redundant with the same field in BinaryInfo. Given that it only exists to pass this information to PostInitializationSetup I think it's fine if it's a parameter to that function.

Reply:
Agreed, changed.